### ISSUE TITLE
AttributeError: module 'torch.jit' has no attribute '_script_if_tracing'

### ISSUE THREAD

**Issue by gnthibault** [pytorch/vision#2514]
Dear all, for some reason, I recently started to have problems with torchvision, although it looks like I haven't changed anything recently.

Do you have any idea what happened ?
Thank you in advance for your help

```
    import segmentation_models_pytorch as smp
  File "/usr/local/lib/python3.6/dist-packages/segmentation_models_pytorch/__init__.py", line 1, in <module>
    from .unet import Unet
  File "/usr/local/lib/python3.6/dist-packages/segmentation_models_pytorch/unet/__init__.py", line 1, in <module>
    from .model import Unet
  File "/usr/local/lib/python3.6/dist-packages/segmentation_models_pytorch/unet/model.py", line 3, in <module>
    from ..encoders import get_encoder
  File "/usr/local/lib/python3.6/dist-packages/segmentation_models_pytorch/encoders/__init__.py", line 4, in <module>
    from .resnet import resnet_encoders
  File "/usr/local/lib/python3.6/dist-packages/segmentation_models_pytorch/encoders/resnet.py", line 28, in <module>
    from torchvision.models.resnet import ResNet
  File "/usr/local/lib/python3.6/dist-packages/torchvision/__init__.py", line 5, in <module>
    from torchvision import models
  File "/usr/local/lib/python3.6/dist-packages/torchvision/models/__init__.py", line 12, in <module>
    from . import detection
  File "/usr/local/lib/python3.6/dist-packages/torchvision/models/detection/__init__.py", line 1, in <module>
    from .faster_rcnn import *
  File "/usr/local/lib/python3.6/dist-packages/torchvision/models/detection/faster_rcnn.py", line 7, in <module>
    from torchvision.ops import misc as misc_nn_ops
  File "/usr/local/lib/python3.6/dist-packages/torchvision/ops/__init__.py", line 1, in <module>
    from .boxes import nms, box_iou
  File "/usr/local/lib/python3.6/dist-packages/torchvision/ops/boxes.py", line 43, in <module>
    @torch.jit._script_if_tracing
AttributeError: module 'torch.jit' has no attribute '_script_if_tracing'
```

Which versions am I using:

> pip freeze | grep torch
> efficientnet-pytorch==0.6.3
> segmentation-models-pytorch==0.1.0
> torch==1.4.0
> torchvision==0.5.0

cc @suo @gmagogsfm

**Comment by fmassa** [comment]
Hi,

You need a more recent version of PyTorch for this to work. We released yesterday torchvision 0.7.0, which is compatible with PyTorch 1.6.0.

Also, looks like there might be some sort of conflict in your system, as your error message mentions `_script_if_tracing`, which was not present in torchvision 0.5.0, so this might be one of the root causes.

I believe this is an issue of version incompatibility, and as such I'm closing this issue, but let us know if you still face those issues after updating PyTorch.

**Comment by cliffplaysdrums** [comment]
I encountered this issue today as well. torch 1.5 was already installed, and I installed torchvision 0.7 via PyCharm globally. I later discovered that this method of install did not result in an upgrade of torch. I initially downgraded torchvision to 0.4 as a workaround. Afterwards a simple `pip3 install torchvision` appropriately upgraded torchvision back to 0.7 _and_ upgraded torch to the compatible 1.6.
